packages/SwingSet/src/vats/comms/clist-inbound.js

```diff
@@ -29,46 +19,43 @@ export function makeInbound(state, stateKit) {
 
   function retireRemotePromiseID(remoteID, rpid) {
     insistRemoteType('promise', rpid);
-    const remote = getRemote(state, remoteID);
-    const lpid = remote.fromRemote.get(rpid);
+    const remote = state.getRemote(remoteID);
+    const lpid = remote.mapFromRemote(rpid);
     assert(lpid, X`unknown remote ${remoteID} promise ${rpid}`);
-    const p = state.promiseTable.get(lpid);
+    const p = state.getPromise(lpid);
```

Comment:
It feels like finer-grained getters (`state.getPromiseSubscribers(lpid)`) could make this more efficient; we don't need to read and deserialize the entire promise record if the caller only cares about one field. Maybe this could be a followup optimization in a future PR.

Reply:
nevermind, I see you did that in the second commit, awesome